app/views/posts/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<% posts.map do |post| %>
+  <br />
+  <% if post.url_post? %>
+    <%= link_to(post.title, post.url) %>
+  <% else %>
+    <%= post.text.truncate(80, {separator: ' '}) %>
+  <% end %>
+  <br />
+  <p>by <%= link_to(post.submitter.name, user_path(post.submitter.name)) %> | posted <%= link_to(time_ago_in_words(post.created_at), post_path(post.id)) %>  ago
```

Comment:
can `post_path(post.id)` be replaced with `post` in this context? at the very least, the `.id` bit can be dropped.